sessionctx/variable/tidb_vars.go

```diff
@@ -124,4 +126,5 @@ const (
 	DefBatchDelete                = false
 	DefCurretTS                   = 0
 	DefDMLBatchSize               = 20000
+	DefInitialIndexJoinBatchSize  = 2048
```

Comment:
2048 is too large, by the way, If this size is larger than max chunk size, it will be reduced to it. See `func (ow *outerWorker) increaseBatchSize() {` for more detail.

Reply:
OK